src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -85,7 +91,36 @@ public async Task RunApplicationAsync(CancellationToken cancellationToken = defa
         {
             AspireEventSource.Instance.DcpModelCreationStop();
         }
+    }
+
+    private async Task StartDashboardAsync(CancellationToken cancellationToken = default)
+    {
+        var dashboardExecutableSpec = new ExecutableSpec();
+        dashboardExecutableSpec.ExecutionType = ExecutionType.Process;
+   
+        // HACK: Just doing this in this branch until the workload mechanism is in place
+        //       and I can emulate it.
+        dashboardExecutableSpec.ExecutablePath = "C:\\Code\\dotnet\\aspire\\artifacts\\bin\\Aspire.Dashboard\\Debug\\net8.0\\Aspire.Dashboard.exe";
+        dashboardExecutableSpec.WorkingDirectory = "C:\\Code\\dotnet\\aspire\\src\\Aspire.Dashboard";
+
+        var environment = new List<EnvVar>();
+        environment.Add(new EnvVar()
```

Comment:
You also need to set the URL of the dashboard itself.

Reply:
Yep. We'll grab these values from the dashboard service host.